content/en/docs/faas/lambda-auto-instrument.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Lambda Auto-Instrumentation
+linkTitle: Lambda Auto-Instrumentation
+weight: 11
+description:
+  Automatically instrument your Lambdas with OpenTelemetry 
+---
+
+The OpenTelemetry community provides standalone instrumentation Lambda layers for the following languages:
+
+* Java
+* JavaScript
+* Python
+
+These can be added to your Lambda using the AWS portal to automatically instrument your application. These layers do not include the Collector which is an optional but recommended standalone Lambda layer.
+
+### Language Requirements
+
+<!-- prettier-ignore -->
+{{< tabpane text=true >}}
+{{% tab Java %}}
+
+The Lambda layer supports the Java 11 (Corretto) Lambda runtime. It does not support the Java 8 Lambda runtimes. For more information about supported Java versions, see the OpenTelemetry Java documentation.
+
+Note: ADOT Lambda Layer for Java Auto-instrumentation Agent - Automatic instrumentation has a notable impact on startup time on AWS Lambda and you will generally need to use this along with provisioned concurrency and warmup requests to serve production requests without causing timeouts on initial requests while it initializes.
+
+<!-- prettier-ignore -->
+{{% /tab %}}
+{{% tab Python %}}
```

Comment:
We can add in each tab what we discussed in today's sig? Regarding supported OTLP exporter by each layer?

We should state that is not necessary to change any environment variable related to the exporters because the defaults work if you have ab embedded collector whose configuration have grpc and http enabled in the OTLP receiver.

The defaults and supported variables by each layer are:

### Java
`OTEL_EXPORTER_OTLP_PROTOCOL=grpc`
Supports: `grpc`, `http/protobuf` and `http/json`
`OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4317`

### Python
`OTEL_EXPORTER_OTLP_PROTOCOL=http/protobuf`
Supports: `http/protobuf` and `http/json`
`OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318`

### NodeJS
OTEL_EXPORTER_OTLP_PROTOCOL env var is not supported. Hard coded exporter use the protocol `http/protobuf`
`OTEL_EXPORTER_OTLP_ENDPOINT=http://localhost:4318`

Reply:
Added a follow on section